plugin-server/src/worker/ingestion/event-pipeline/embedErrorEventStep.ts

```diff
@@ -0,0 +1,92 @@
+import { FeatureExtractionPipeline, FeatureExtractionPipelineOptions } from '@xenova/transformers'
+
+import { defaultConfig } from '../../../config/config'
+import { runInstrumentedFunction } from '../../../main/utils'
+import { PreIngestionEvent } from '../../../types'
+import { status } from '../../../utils/status'
+import { EventPipelineRunner } from './runner'
+
+const errorEmbeddingModel = defaultConfig.ERROR_EVENT_EMBEDDING_MODEL
+const errorEmbeddingEnabledTeams = (defaultConfig.ENABLE_ERROR_EMBEDDING_TEAM_IDS || '')
+    .split(',')
+    .map((id) => parseInt(id, 10))
```

Comment:
Will these consts be properly updated when the envvar changes? It's better to pass PluginServerConfig as a parameter to `initEmbeddingModel`

Reply:
i thought the pods restarted when we set env vars anyway

Reply:
`export const defaultConfig = overrideWithEnv(getDefaultConfig())`, so it indeed works. Up until we refactor that bit and forget about this direct access.